### PR TITLE
[runtime] Move mono_get_context_capture_method () from security-manager.c to object.c.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

File: mono/metadata/object.c [hunk 1]
```diff
@@ -8008,6 +8008,25 @@ mono_wait_handle_get_handle (MonoWaitHandle *handle)
...
 
+/*
+ * Returns the MonoMethod to call to Capture the ExecutionContext.
+ */
+MonoMethod*
+mono_get_context_capture_method (void)
+{
+	static MonoMethod *method;
+
+	/* older corlib revisions won't have the class (nor the method) */
```
Comment (spouliot) [inline mono/metadata/object.c]: that comment does not make sense anymore and the check was not moved, which is good :-)

**Pull request merged**